docs/sources/administration/configuration.md

```diff
@@ -840,122 +1174,124 @@ If the remote HTTP image renderer service runs on a different server than the Gr
 ### concurrent_render_request_limit
 
 Concurrent render request limit affects when the /render HTTP endpoint is used. Rendering many images at the same time can overload the server,
-which this setting can help protect against by only allowing a certain amount of concurrent requests.
+which this setting can help protect against by only allowing a certain amount of concurrent requests. Default is `30`.
 
 ## [panels]
 
+### enable_alpha
+
+Set to `true` if you want to test alpha panels that are not yet ready for general usage. Default is `false`.
+
 ### disable_sanitize_html
 
-If set to true Grafana will allow script tags in text panels. Not recommended as it enable XSS vulnerabilities. Default
-is false. This settings was introduced in Grafana v6.0.
+If set to true Grafana will allow script tags in text panels. Not recommended as it enable XSS vulnerabilities. Default is false. This settings was introduced in Grafana v6.0.
 
 ## [plugins]
 
 ### enable_alpha
 
-Set to true if you want to test alpha plugins that are not yet ready for general usage.
+Set to `true` if you want to test alpha plugins that are not yet ready for general usage. Default is `false`.
 
 ### allow_loading_unsigned_plugins
 
 Enter a comma-separated list of plugin identifiers to identify plugins that are allowed to be loaded even if they lack a valid signature.
 
-## [feature_toggles]
-### enable
+<hr>
 
-Keys of alpha features to enable, separated by space. Available alpha features are: `transformations`
+## [plugin.grafana-image-renderer]
 
-## [tracing.jaeger]
+For more information, refer to [Image rendering]({{< relref "image_rendering.md" >}}).
 
-Configure Grafana's Jaeger client for distributed tracing.
+### rendering_timezone
```

Comment:
Have plans to add these to the image rendering instead. Don't want to have them duplicated. Fine for now